### PR TITLE
OPAE: Fix mmap mmio region

### PR DESCRIPTION
**Pull request by anandaravuri**
Changes:
 -Add mmap mmio region address to workspace list

### CODE REVIEW

File: libopae/src/mmio.c [hunk 1]
```diff
@@ -166,7 +166,7 @@ static fpga_result map_mmio_region(fpga_handle handle, uint32_t mmio_num)
 	wsid = wsid_gen();
 	if (!wsid_add(&_handle->mmio_root,
 		      wsid,
-		      (uint64_t) NULL,
+		      (uint64_t) addr,
```
Comment (tswhison) [inline libopae/src/mmio.c]: Do we also need to add the iova, or is that done somewhere else?

**Comment by rpSTTD** [comment]
iova is added in fpgaPrepareBuffer() API.

File: libopae/src/mmio.c (hunk 1 above)
Comment (anandaravuri) [reply libopae/src/mmio.c]: Adding MMIO pointer to work space list , ivoa is not valid in mmio, this not buffer pointer

Comment (luebbers) [reply libopae/src/mmio.c]: For MMIO, that doesn't seem necessary (no need for user space or the accelerator to know that).

**Pull request merged**